docs/layers/language-server-protocol.md

```diff
@@ -79,6 +79,14 @@ npm install -g javascript-typescript-langserver
 pip install --user python-language-server
 ```
 
+**julia:**
+
+The `LanguageServer` package must be installed in Julia (0.6 or greater), i.e.
+
+```sh
+julia> Pkg.clone("https://github.com/JuliaEditorSupport/LanguageServer.jl")
```

Comment:
with new package system in Julia 0.7 and above, we have a package mode in Julia REPL. 
in REPL, hit "]" to enter the package management mode, then `add LanguageServer` to install the package.

Reply:
The installation step is from LanguagerServer's wiki,  ok, I will add this.